README.md

```diff
@@ -983,6 +984,105 @@ This section describes code style for [ECMAScript 2015 Language Specification](h
 
 [⬆ back to TOC](#table-of-contents)
 
+### Arrow functions
+
+* [Arrow function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)
+  should be used where an anonymous function is expected (when you need function and you don't want bind it to an
+  identifier):
+
+  > Explanation:
+  > * `this` keyword in arrow function binds to the upper scope's `this`. It prevents usage invalid `this` or usage less
+  >    efficient `Function.prototype.bind` function.
+  > * The notation is shorter than function expression.
+
+  ```js
+  [1, 2, 3].map((x) => {
+      // ...
+  });
+
+  // Use `function` here, because pass `someObj` as `this` argument.
+  [1, 2, 3].map(function (x) {
+      // ...
+  }, someObj);
+  ```
+
+* If you bind function with some name, use `function declaration`:
```

Comment:
I can only think of several reasons, although they are not very convincing ones:
1) It looks simpler and cleaner. When you see a named function declaration, you know that's a function right away without looking at the assignment
2) You can see the function name in an exception's call stack (vs. `Anonymous function` for unnamed declaration)
3) Functions can be used for various functional programming techniques for which you will need the full arsenal of `Function.prototype`. Arrow functions can not be bound to other `this` for example.

IDK, call me conservative but I think `function <name>(<params...>) {<body>}` just _looks_ better.